phylogenetic/defaults/config.yaml

```diff
@@ -28,7 +28,7 @@ ancestral:
   inference: "joint"
 
 traits:
-  columns: "country"
+  columns: "country host"
```

Comment:
_question_

Does it make sense to infer the `host` trait when the tree is heavily biased towards human hosts? To me, it looks like Lassa spilled over from humans to mice in the [staging tree](https://next.nextstrain.org/staging/lassa/l?c=host), but maybe I'm just interpreting it wrong?

Reply:
My understanding is that Lassa tends to be sequenced as part of an outbreak investigation (when it spills over into humans) which is why human hosts are overrepresented in the tree. However, the reservoir species is known to be the rat `Mastomys natalensis` (from [the cdc site](https://www.cdc.gov/lassa-fever/about/index.html#:~:text=Lassa%20fever%20is%20a%20severe,known%20to%20carry%20Lassa%20virus.)).

> infer the host trait 

Ah, perhaps you're suggesting only annotating the tips for "host" and not inferring ancestral traits of "host" up the tree since it may be misleading...perhaps like https://nextstrain.org/lassa/l?c=host_species ? 

Inferring the trait can help incentivize more sequencing... but I can see your point about the interpretation. I believe coloring only the tips by host would require using `augur export --color-by-metadata`?

Reply:
> Ah, perhaps you're suggesting only annotating the tips for "host" and not inferring ancestral traits of "host" up the tree since it may be misleading...perhaps like https://nextstrain.org/lassa/l?c=host_species ?
> 
> Inferring the trait can help incentivize more sequencing... but I can see your point about the interpretation. I believe coloring only the tips by host would require using augur export --color-by-metadata?

Yup, I think it would be less misleading to just color the tips by host. This would already be done with the addition of `host` to the auspice_config.json as you've included in 5a31b10387472b25bb228c21aab8b51fbc30e1ff.